features/product/viewing_products/access_simple_product_show_page_from_products_index.feature

```diff
@@ -0,0 +1,90 @@
+@viewing_products
```

Comment:
Think about steps unification here, since some of them do not contain `is`, like:

`Then I should see product's code "123456789"`

while the others do: 

`And I should see product's depth is 20`